### ISSUE TITLE
Installing chart elasticsearch fails

### ISSUE THREAD

**Issue by almontasser** [mosip/mosip-infra#483]
```
TASK [helm : Install chart elasticsearch] *******************************************************************************************************************************
fatal: [console.sb]: FAILED! => {"changed": true, "cmd": ["/home/mosipuser/bin/helm", "--kubeconfig",
"/home/mosipuser/.kube/mzcluster.config", "install", "elasticsearch", 
/home/mosipuser/mosip-infra/deployment/sandbox-v2/helm/charts/elasticsearch", "-f",
 "/home/mosipuser/mosip-infra/deployment/sandbox-v2/helm/charts/elasticsearch/values.yaml", "-n", "monitoring", 
"--create-namespace", "--set-string", "dummy=value,", "--wait", "--timeout", "20m0s"], "delta": "0:20:00.895887", 
"end": "2020-12-28 16:27:08.408529", "msg": "non-zero return code", "rc": 1, "start": "2020-12-28 16:07:07.512642", 
"stderr": "Error: timed out waiting for the condition", "stderr_lines": ["Error: timed out waiting for the condition"], "stdout": "",
"stdout_lines": []}

PLAY RECAP ***************************************************************************************************************
console.sb                 : ok=150  changed=85   unreachable=0    failed=1    skipped=23   rescued=0    ignored=0   
dmzmaster.sb               : ok=12   changed=9    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
dmzworker0.sb              : ok=12   changed=9    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
mzmaster.sb                : ok=50   changed=37   unreachable=0    failed=0    skipped=28   rescued=0    ignored=1   
mzworker0.sb               : ok=36   changed=26   unreachable=0    failed=0    skipped=27   rescued=0    ignored=0   
mzworker1.sb               : ok=36   changed=26   unreachable=0    failed=0    skipped=27   rescued=0    ignored=0   
mzworker2.sb               : ok=37   changed=27   unreachable=0    failed=0    skipped=27   rescued=0    ignored=0
```

**Comment by almontasser** [comment]
the output of helm install --debug:


[debug] CHART PATH: /home/mosipuser/mosip-infra/deployment/sandbox-v2/helm/charts/elasticsearch\n\nclient.go:108: [debug] creating 1 resource(s)\nclient.go:108: [debug] creating 5 resource(s)\nwait.go:53: [debug] beginning wait for 5 resources with timeout of 20m0s\nwait.go:334: [debug] StatefulSet is not ready: monitoring/elasticsearch-master. 0 out of 1 expected pods are ready\nwait.go:334: [debug] StatefulSet is not ready: monitoring/elasticsearch-master. 0 out of 1 expected pods are ready\nwait.go:334:

**Comment by abdihakx** [comment]
@almontasser How did you fix this issue?

**Comment by almontasser** [comment]
@abdihakx My problem was slow internet. So I've changed the timeout from 20m to 120m in the file: `roles/helm/tasks/main.yml`.
It also could happen because of low RAM.

**Comment by abdihakx** [comment]
@almontasser I tried changing the timeout from 20m to 120m in `roles/helm/tasks/main.yml` as specified in #503 but this did not help. It only made it took longer before the error was thrown. I am not sure if the issue is internet coz last I checked the server had 50Mbps+ speed. For RAM, I assigned the recommended RAM of 16GB on the console VM (This is where is thrown). But the other VMs have a lower RAM than the recommended (But I guess this should not be an issue since I am deploying a test sandbox on-prem).
